addons/web/static/src/views/form/form_controller.scss

```diff
@@ -291,11 +293,23 @@
     .alert {
         --alert-padding-y: #{map-get($spacers, 2)};
 
+        &:nth-last-child(1 of .alert) {
+            --alert-margin-bottom: #{map-get($spacers, 2)};
+        }
+
         .o_field_widget, .o_field_widget p {
             margin-bottom: 0;
         }
     }
 
+    .o_form_sheet {
+        --alert-margin-bottom: initial;
+
+        .alert:nth-last-child(1 of .alert) {
+            --alert-margin-bottom: initial;
+        }
+    }
+
```

Comment:
To set the correct margin for **alerts placed above the sheet**, you first define a margin for **all alerts**, then reset the margin for alerts inside the sheet 🤔 .

It might be cleaner to target alerts outside the sheet directly.
Something like this:
```scss
.o_form_sheet_bg > .alert {
    --alert-margin-bottom: #{map-get($spacers, 1)};

    &:nth-last-child(1 of .alert) {
        --alert-margin-bottom: #{map-get($spacers, 2)};
    }
}
```